cirq/circuits/circuit_test.py

```diff
@@ -3817,3 +3818,49 @@ def test_zip():
             cirq.Circuit(cirq.X(a), cirq.CNOT(a, b)),
             cirq.Circuit(cirq.X(b), cirq.Z(b)),
         )
+
+
+def test_repr_html_escaping():
+
+    class TestGate(cirq.Gate):
+
+        def __init__(self, n_qubits, label):
+            self.n_qubits = n_qubits
+            self.label = label
+
+        def num_qubits(self):
+            return self.n_qubits
+
+        def _circuit_diagram_info_(self, args):
+            return cirq.CircuitDiagramInfo(wire_symbols=[self.label] *
+                                           self.n_qubits)
+
+    F2 = TestGate(2, "< ' F ' > ")
+    a, b = cirq.LineQubit.range(2)
+    c = cirq.NamedQubit("|c>")
+
+    circuit = cirq.Circuit([F2(a, b), F2(b, c), F2(a, c)])
+    original_diagram = """
+0: ─────< ' F ' > ────────────────< ' F ' > ───
+        │                         │
+1: ─────< ' F ' > ───< ' F ' > ───┼────────────
+                     │            │
+|c>: ────────────────< ' F ' > ───< ' F ' > ───
+""".strip()
+
+    escaped_diagram = ('<pre style="overflow: auto; '
```

Comment:
This is a little bit fragile to test against the full output string: for example, if we change the style tag, this test will have to change, which even though it is focusing on an unrelated logic. I would just simply assert that `_repr_html` contains `&lt; &#x27; F &#x27; &gt;`.

Reply:
Got it! Changing it.

Reply:
Hey @balopat please have a look at commit [a45f2a0](https://github.com/quantumlib/Cirq/pull/3221/commits/a45f2a050e28c8ad1e9d7405e0820f9cd4c44c36).